README.md

```diff
@@ -89,7 +89,8 @@ Information on how to get started can be found in our
 [Hot reload animation]: https://github.com/flutter/website/blob/main/src/assets/images/docs/tools/android-studio/hot-reload.gif?raw=true
 [Hot reload]: https://flutter.dev/docs/development/tools/hot-reload
 [Visual Studio Code]: https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
-[IntelliJ / Android Studio]: https://plugins.jetbrains.com/plugin/9212-flutter
+[IntelliJ]: https://plugins.jetbrains.com/plugin/9212-flutter
+[Android Studio]: https://flutter.dev/docs/development/tools/android-studio
```

Comment:
should this point to https://docs.flutter.dev/get-started/editor?tab=androidstudio instead?

Reply:
Yes